portlet-tck_3.0/driver/src/main/java/javax/portlet/tck/driver/TCKLiferayTestDriver.java

```diff
@@ -232,7 +232,10 @@ protected void click(WebElement wel) {
           url = wel.getAttribute("href");
 
 	      if ("a".equals(tagName)) {
-             if (url.contains("v3headerportlettests")) {
+             if (url.contains("v3headerportlettests".toLowerCase())) {
```

Comment:
@vsingleton `"v3headerportlettests".toLowerCase()` is a no-op since it is already lower case. Did you mean to do `url.toLowerCase().contains()` instead? In the code review commit I will remove the no-op.